src/containers/quiz/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx

```diff
@@ -0,0 +1,59 @@
+import { ChangeEvent, FC } from 'react'
+
+import Box from '@mui/material/Box'
+import { SxProps } from '@mui/material/styles'
+
+import QuizQuestion from '~/containers/quiz/quiz-question/Question'
+import { spliceSx } from '~/utils/helper-functions'
+import { styles } from '~/containers/quiz/scroll-question-quiz-view/ScrollQuestionsQuizView.styles'
+
+import { Question, UseFormEventHandler } from '~/types'
+
+export interface QuizViewProps {
+  questions: Question[]
+  handleInputChange: UseFormEventHandler<
+    Record<string, string | string[]>,
+    ChangeEvent<HTMLInputElement>
+  >
+  handleNonInputValueChange: (key: string) => (value: string | string[]) => void
+  answers: Record<string, string | string[]>
+  isEditable?: boolean
+  showCorrectAnswers?: boolean
+  showPoints?: boolean
+  showAnswersCorrectness?: boolean
```

Comment:
Same here

Reply:
Thank you for noticing. I guess properties with `show` prefix already suit well for these cases because these names are both short and reflect boolean type.